server/chat-commands/room-settings.ts

```diff
@@ -1299,6 +1299,52 @@ export const commands: ChatCommands = {
 	removeroomaliashelp: [
 		`/removeroomalias [alias] - removes the given room alias of the room the command was entered in. Requires: & ~`,
 	],
+
+	resettierdisplay: 'roomtierdisplay',
+	roomtierdisplay(target, room, user, connection, cmd) {
```

Comment:
Was this a mistake? It should probably go in a different PR than a TS refactor.

Reply:
I asked you about this in #sim-dev and you said it was fine, unless I misinterpreted something.

Reply:
I don't remember this at all. Can you move it to a separate PR now, though?